src/components/BreadCrumb.vue

```diff
@@ -0,0 +1,48 @@
+<script setup lang="ts">
+import BaseButton from "./base/BaseButton.vue";
+import LeftArrow from "@/assets/icons/long-arrow-left.svg?component";
+
+export interface Props {
+  backPath: string;
+  firstTier: string;
+}
+
+const props = withDefaults(defineProps<Props>(), {
+  backPath: "/",
+  firstTier: "",
+});
+</script>
+
+<template>
+  <div class="breadcrumbs">
+    <BaseButton
+      button-type="breadcrumb"
+      button-size="slim"
+      :to="props.backPath"
+    >
+      <LeftArrow />
+      Back
+    </BaseButton>
+    <BaseButton
+      button-type="breadcrumb-rev"
+      button-size="slim"
+      :to="props.firstTier"
```

Comment:
small note: you don't need to reference props via `props` within template tag. They should be exposed automatically so this could be just
```suggestion
      :to="firstTier"
```

Reply:
amazing - i do get elint warning about unused var on props declaration?

Reply:
If props is not used in `script` tag you can simply get rid of the variable and keep just `defineProps...` :)